src/lib.rs

```diff
@@ -124,8 +124,6 @@ mod functional_test;
 #[macro_use]
 mod macros;
 
-pub use chrono;
```

Comment:
I want to keep the reexport. Let me explain in the ticket.

Reply:
https://github.com/quickwit-oss/tantivy/pull/1307/commits/c8718ceecf433f506ab012f9da3f81bf10d9d73b